src/BusinessTable.jsx

```diff
@@ -0,0 +1,151 @@
+import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import { Table, Thead, Tbody, Tr, Th, Td } from "@chakra-ui/react";
+
+const BusinessTable = () => {
+  const { backend } = useBackend();
+  const [data, setData] = useState([]);
+
+  const getData = async () => {
+    try {
+      const response = await backend.get('/business');
+      setData(response.data);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  };
+
+  useEffect(() => {
+    getData();
+  });
+
+  return (
+    <div>
+      <Table variant="striped" colorScheme="teal">
+        <Thead>
+          <Tr>
+            <Th>Id</Th>
+            <Th>Type</Th>
+            <Th>Name</Th>
+            <Th>Street</Th>
+            <Th>Zipcode</Th>
+            <Th>State</Th>
+            <Th>Qb Vendor Name</Th>
+            <Th>Qb City State Zip</Th>
+            <Th>Primary Phone</Th>
+            <Th>Backup Phone</Th>
+            <Th>Primary Email</Th>
+            <Th>Comments</Th>
+            <Th>Fax phone</Th>
+            <Th>Contact name</Th>
+            <Th>Website</Th>
+            <Th>Business hours</Th>
+            <Th>Find out</Th>
+            <Th>Onboarding status</Th>
+            <Th>Join date</Th>
+            <Th>Input type status</Th>
+            <Th>Vendor type</Th>
+            <Th>Status</Th>
+            <Th>Pets of the homeless discount</Th>
+            <Th>Updated by</Th>
+            <Th>Updated date time</Th>
+            <Th>Sync to qb</Th>
+            <Th>Veterinary</Th>
+            <Th>Ressoure</Th>
+            <Th>Food</Th>
+            <Th>Donation</Th>
+            <Th>Family shelter</Th>
+            <Th>Wellness</Th>
+            <Th>Spray neuter</Th>
+            <Th>Financial</Th>
+            <Th>Re home</Th>
+            <Th>Er boarding</Th>
+            <Th>Senior</Th>
+            <Th>Cancer</Th>
+            <Th>Dog</Th>
+            <Th>Cat</Th>
+            <Th>Fph phone</Th>
+            <Th>Contact phone</Th>
+            <Th>Web notes</Th>
+            <Th>Internal notes</Th>
+            <Th>Published</Th>
+            <Th>Shelter</Th>
+            <Th>Domestic Violence</Th>
+            <Th>Web Date Init</Th>
+            <Th>Ent Qb</Th>
+            <Th>Service Request</Th>
+            <Th>Inactive</Th>
+            <Th>Final Check</Th>
+            <Th>Created By</Th>
+            <Th>Created Date</Th>
+            <Th>City</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {data.map((item, index) => (
+            <Tr key={index}>
+              <Td>{item.id}</Td>
+              <Td>{item.type}</Td>
+              <Td>{item.name}</Td>
+              <Td>{item.street}</Td>
+              <Td>{item.zipcode}</Td>
+              <Td>{item.state}</Td>
```

Comment:
good, however same issue here with having 55 lines. see if you can iterate over the attributes within item and map it onto the table accordingly. if the variables are not outputting in the correct order, then you can also create a static variable at the top to indicate what order of attributes you want to iterate over and use that to come with a loop to create all the data entries for the columns in a row